runtime/mods/kiroku/mod.hcl

```diff
@@ -0,0 +1,8 @@
+mod {
+    name = "kiroku"
+    author = "Ruin0x11"
+    version = "0.0.1"
+    description = "Adds an item which displays the total number of creatures the player has killed."
+
+    dependencies = ["core"]
```

Comment:
This `dependencies = ["core"]` is needed in the current implementation or written as just an example? I think it's better to load "core" first regardless of whether there are some mods depending on "core" or not (in other words, dependency on core is specified explicitly or not), although it requires to hard-code. Probably all mods depend on core and won't work without that.

Reply:
Technically mods can still be loaded if they do not depend on `core`, it's just that there is no guarantee that anything inside `core` will be loaded when the mod is loaded. It is entirely possible for mods to not use anything from `core`, so I preferred to make the relationship explicit for now. There should really be a separation of `core` into two mods, `core` for anything the native code relies on in a hardcoded manner, and `elona`, which contains the base game content. In this case `core` would be the implicit dependency, and `elona` would be able to be removed with no adverse effects. However, it would be difficult to do since there is currently a lot of hardcoding of both types of content.

Reply:
I see, agree with that.